README.md

```diff
@@ -52,7 +52,7 @@ Below are the major frameworks utilized for the creation of this project.
 ## Getting Started
 
 **Instructions on how to access my project locally:** 
-- Enter this url: [https://github.com/OkoyeCharles/todo-list-app](https://github.com/OkoyeCharles/todo-list-app) in your web browser.
+- Enter this url: [https://github.com/OkoyeCharles/todo-app](https://github.com/OkoyeCharles/todo-app) in your web browser.
 - Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
 - Select download Zip option from drop down menu.
 - Once the download is complete you will be able to access my project locally . 
```

Comment:
Great job on specifying how users can access your repository. however, I kindly suggest you consider other ways on how users can get the repository into their local machine,  like `git clone repository name`. Giving other options for accessing the repo will increase the chance of users contributing to your repository. 🤟